Objects/descrobject.c

```diff
@@ -78,8 +78,8 @@ descr_check(PyDescrObject *descr, PyObject *obj, PyObject **pres)
     }
     if (!PyObject_TypeCheck(obj, descr->d_type)) {
         PyErr_Format(PyExc_TypeError,
-                     "descriptor '%V' for '%s' objects "
```

Comment:
Why has this been changed?

There was a difference with classmethod descriptors, but classmethod descriptors were special, they are applied to types, not instances.

Reply:
This message is for method, not classmethod.  But I changed classmethod error message too.
I feel same error message makes sense for classmethod too.
Discuss about actual error message on bpo.